### PR TITLE
ci: re-enable manifests repo

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: .github/workflows/build-main.yaml [hunk 1]
```diff
@@ -58,72 +58,75 @@ jobs:
...
+
+          cd ${{ env.PROJECTS }}/sco-manifests
+
+          ls -lR connectors
+
+          git config user.email "sco-connectors@sco1237896.github.com"
+          git config user.name "sco-connectors"
+
+          git checkout -b ${BRANCH_NAME}
+          git add connectors/sco-connector
```
Comment (asmigala) [inline .github/workflows/build-main.yaml:112]: The pipeline fails here with `fatal: pathspec 'connectors/sco-connector' did not match any files`. 

The `connectors` directory at that point has the following contents:

```
| drwxr-xr-x. 1 root root     544 Sep  6 07:16 connector-catalog-aws
| drwxr-xr-x. 1 root root     432 Sep  6 07:17 connector-catalog-azure
| drwxr-xr-x. 1 root root     334 Sep  6 07:19 connector-catalog-gcp
| drwxr-xr-x. 1 root root      28 Sep  6 07:19 connector-catalog-itops
| drwxr-xr-x. 1 root root     262 Sep  6 07:19 connector-catalog-messaging
| drwxr-xr-x. 1 root root      58 Sep  6 07:20 connector-catalog-misc
| drwxr-xr-x. 1 root root     228 Sep  6 07:21 connector-catalog-nosql
| drwxr-xr-x. 1 root root     400 Sep  6 07:22 connector-catalog-saas
| drwxr-xr-x. 1 root root     164 Sep  6 07:22 connector-catalog-social
| drwxr-xr-x. 1 root root     452 Sep  6 07:23 connector-catalog-sql
| drwxr-xr-x. 1 root root     220 Sep  6 07:23 connector-catalog-storage
| -rw-r--r--. 1 root root 1919348 Sep  6 07:23 connectors.json
```

Looking at the old cos-manifests repo, I assume the intention here is to have a subdirectory that would contain all the connector categories?

Comment (lburgazzoli) [reply .github/workflows/build-main.yaml:112]: for rhoc we had subdirectory for camel and debezium conenctors, here we don't so there is not more need to have a subdirectory

**Comment by asmigala** [comment]
@lburgazzoli after fixing the subdirectory issue mentioned above, the pipeline passes locally when run through `act`, _except_ for the `connector-mongodb-it` tests. However, those tests pass locally on their own (not through `act`), so I assume the problem might be in the image used by `act`. I'd say we merge this and see if the mongodb tests pass when run as actual github action.

**Comment by lburgazzoli** [comment]
@rinaldodev @valdar can you have a look ?

**Review by rinaldodev — APPROVED** [review]

**Comment by asmigala** [comment]
The PR workflow still needs fixing, currently having trouble running it locally

**Comment by lburgazzoli** [comment]
@asmigala can we do fixes in a subsequent pr ?

**Pull request merged**